client/src/PlayerLanding/PlayerRole.tsx

```diff
@@ -16,44 +17,58 @@ interface PlayerRoleProps {
 function PlayerRole({ role }: PlayerRoleProps) {
   const [player] = usePlayer();
   const [isHolding, setIsHolding] = useState(false);
+  const [hasSeenRole, setHasSeenRole] = useState(false);
```

Comment:
I think this is pretty overcomplicated, can we simplify to just click the thumbprint, and then acknowledge that they've seen it?

Reply:
We don't need to only show while holding their thumb on it